sycl/include/sycl/vector_preview.hpp

```diff
@@ -601,6 +601,11 @@ class vec : public detail::vec_arith<DataT, NumElements> {
       return static_cast<RetType>(m_Data[Index]);
   }
 
+  // Required by swizzles when working with sycl::vec pointers.
```

Comment:
Can we add an in-tree test for this instead of relying on out-of-tree CTS to catch it?

Reply:
Sure.

Reply:
Thanks for adding the test!